src/CFamily/PreCompiledHeaders/PreCompiledHeadersEventListener.cs

```diff
@@ -29,7 +29,7 @@
 
 namespace SonarLint.VisualStudio.Integration.Vsix.CFamily
 {
-    interface IPreCompiledHeadersEventListener : IDisposable
+    public interface IPreCompiledHeadersEventListener : IDisposable
```

Comment:
This is instantiated by the SonarLintDaemon package so it needs to be public now.